app/forms/field_handlers/field_handler.py

```diff
@@ -43,11 +48,19 @@ def label(self):
     def guidance(self):
         return self.answer_schema.get("guidance", "")
 
+    @staticmethod
+    def format_question_title(error_message, question_title):
```

Comment:
The method name would be clearer as something like `interpolate_question_title_into_error` or `insert_question_title_into_error`.